spynnaker/pyNN/models/neuron/connection_holder.py

```diff
@@ -78,6 +78,10 @@ def __init__(
             This should accept a single parameter, which will contain the
             data requested
         :type notify: callable(ConnectionHolder, None) or None
+        :param app_edge:
```

Comment:
dont exist in the call. redundant docs

Reply:
I wonder why I added it.  Oh well, gone now